### PR TITLE
Back out changes to add new info metrics for 6.0.1

### PR DESCRIPTION
**Pull request by mason359**
This branch will be used to make the release for 6.0.1. I made the minimal changes to the dashboard for it to function in 6.0.1, which means the new formatting changes and fixes that were added for 6.0.2 will be included in this 6.0.1 release.
Screen Shot 2023-04-04 at 3 48 01 PM
Screen Shot 2023-04-04 at 3 48 13 PM

### CODE REVIEW

File: grafana/provisioning/dashboards/qumulo/cluster.json [hunk 1]
```diff
@@ -70,6 +70,30 @@
...
+          },
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "Cluster Name"
+            },
+            "properties": [
+              {
+                "id": "custom.width",
+                "value": 283
```
Comment (vtereso) [inline grafana/provisioning/dashboards/qumulo/cluster.json:82]: Just arbitrary values I guess

File: grafana/provisioning/dashboards/qumulo/node.json [hunk 2]
```diff
@@ -318,53 +342,39 @@
           },
           "editorMode": "builder",
           "exemplar": false,
-          "expr": "last_over_time(qumulo_node_info{instance=~\"$cluster\"}[24h])",
```
Comment (vtereso) [inline grafana/provisioning/dashboards/qumulo/node.json:321]: Why are these bits removed?

**Review by vtereso — COMMENTED** [review]
Some Qs about where these changes are coming from? Can you link me to the 6.0.2 reference branch?

File: grafana/provisioning/dashboards/qumulo/cluster.json (hunk 1 above)
Comment (mason359) [reply grafana/provisioning/dashboards/qumulo/cluster.json:82]: Yeah, I changed the width of a couple of the panels bc they have less information in them now

File: grafana/provisioning/dashboards/qumulo/node.json (hunk 2 above)
Comment (mason359) [reply grafana/provisioning/dashboards/qumulo/node.json:321]: This was the node model field, which we didn't add until 6.0.2

**Comment by mason359** [comment]
> Some Qs about where these changes are coming from? Can you link me to the 6.0.2 reference branch?

The 6.0.2 branch is just main, we haven't made any changes for 6.0.3 yet

**Comment by vtereso** [comment]
LGTM

**Pull request merged**